internal/target/list_response.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package target
+
+import (
+	"github.com/hashicorp/boundary/internal/refreshtoken"
+)
+
+// will likely eventually become a generic response and moved from target
```

Comment:
Can we turn this into a TODO comment and then add proper godoc for this since it is exported?